composer.json

```diff
@@ -35,6 +35,8 @@
     },
     "scripts": {
         "test": "XDEBUG_MODE=coverage phpunit --coverage-clover build/logs/clover.xml",
+        "coverage-html": "XDEBUG_MODE=coverage phpunit --coverage-html build/logs/html",
+        "multicover": "./devtools/php-multicover.sh",
```

Comment:
hm isn't that script MacOS dependent? what about linux dev environments? Shall we really go that path?

Reply:
I don't know about Linux right now. It was really just a waste product of my efforts getting hold of the coverage. In normal situations, it is perfectly fine to run tests on a single version of PHP.

This whole process of merging coverage reports can't be streamlined anyway because `phpunit-merger` isn't available for PHP8 yet.

So, in essence, this is really just a helper script aiding to help people go down the rabbit hole, when actually required.

Reply:
The program should now be compatible with both operating systems, see 6a06e6d52.